src/settings/MappingProfiles/MappingProfilesForm.js

```diff
@@ -467,10 +453,12 @@ MappingProfilesFormComponent.propTypes = {
 const mapStateToProps = state => {
   const okapi = state.okapi || null;
   const mappingDetails = state.form[formName]?.values.profile?.mappingDetails || {};
+  const mappingMarcFieldProtectionSettings = state.form[formName]?.values.profile?.marcFieldProtectionSettings || [];
```

Comment:
Can it be renamed to `mappingMarcFieldProtectionFields` as far as it is used just for `MARCDetailsProps`?